services/MerchStoreService.ts

```diff
@@ -540,4 +540,12 @@ export default class MerchStoreService {
       await orderPickupEventRepository.deletePickupEvent(pickupEvent);
     });
   }
+
+  public async getCartItems(items: string[]): Promise<Map<string, PublicMerchItemOption>> {
+    return this.transactions.readOnly(async (txn) => {
+      const merchItemOptionRepository = Repositories.merchStoreItemOption(txn);
+      const itemOptionModels = await merchItemOptionRepository.batchFindByUuid(items);
+      return new Map(items.map((option) => [option, itemOptionModels.get(option).getPublicMerchItemOption(false)]));
+    });
```

Comment:
Like we discussed in our earlier meeting, let's return an array instead of a map that maintains the same order as the items provided - that way the response is cleaner and the frontend can just take our response and display it without having to index through the map to get options.